meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,297 @@
+  ],
+  "query": "",
+  "vector": [0, 1, 2],
+  "processingTimeMs": 0,
+  "limit": 20,
+  "offset": 0,
+  "estimatedTotalHits": 2
+}
+```
+
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/product/discussions/677).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
+
+Done by @Kerollmops in #3825
+
+
+## Display ranking scores at search
+
+Meilisearch v1.3.0 introduces ranking scores per document via the `showRankingScore` search query parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScore": true }'
+```
+
+Using this search query parameter means that each document will return with a `_rankingScore` property displaying a score between 0 and 1. The higher the ranking score, the more relevant the document.
+
+```json 
+"_rankingScore": 0.8575757575757575,
+```
+
+### Ranking score details - Experimental 
+
+Additionally, as an experimental feature, you can view detailed scores per ranking rule for each document via the `showRankingScoreDetails` search query parameter. 
+
+```bash 
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "Batman Returns", "showRankingScoreDetails": true }'
+```
+
+With this search query parameter in place each document will return with an `_rankingScoreDetails` JSON object. That object will contain a property and a score value for each ranking rule. 
+
+```json 
+"_rankingScoreDetails": {
+  "words": {
+    "order": 0,
+    "matchingWords": 1,
+    "maxMatchingWords": 1,
+    "score": 1.0
+  },
+  "typo": {
+    "order": 1,
+    "typoCount": 0,
+    "maxTypoCount": 1,
+    "score": 1.0
+  },
+  "proximity": {
+    "order": 2,
+    "score": 1.0
+  },
+  "attribute": {
+    "order": 3,
+    "attributes_ranking_order": 0.8,
+    "attributes_query_word_order": 0.6363636363636364,
+    "score": 0.7272727272727273
+  },
+  "exactness": {
+    "order": 4,
+    "matchType": "noExactMatch",
+    "score": 0.3333333333333333
+  }
+}
+```
+
+This experimental feature can be turned on via the HTTP API using v1.3.0's new `/experimental-features` endpoint. 
+
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "scoreDetails": true
+  }'
+
+```
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/product/discussions/674).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
```

Comment:
Could we remove the "Be careful if using it in production"? I feel the score is production-ready, and we want as many people as possible to test it, so we shouldn't discourage production use.

A more accurate statement would be "Code using this feature might need to be modified in future versions".

Reply:
tbh I'm not sure I 💯 understand the messaging from https://github.com/meilisearch/engine-team/blob/main/resources/experimental-features.md now that I re-read it.

On the one hand: 

> ⚠️ Experimental features are NOT [prototypes](https://github.com/meilisearch/engine-team/blob/main/resources/prototypes.md). All experimental features are thoroughly tested before release and follow the same quality standards as other features.

On the other hand:

> Experimental features are not recommended for production environments

Why?

I understand not recommending Prototypes for production environments because they could exhibit quality of implementation issues, but for experimental features I think the trade off is more along the line: "you can use it if you accept that you might have to do more work to upgrade during versions".

What do you think?

Reply:
No strong opinion on it! 
The message I want to see is "careful, the API can break". 

For me, in the guide it should have been written something like "Experimental features are not recommended for production environments because the API can break between 2 versions of Meilisearch"
You can re-phrase it differently if you are not comfortable with it. The main purpose is to let the users know the API can break.

Reply:
"⚠️ The experimental feature API can break between Meilisearch versions. Be aware of this if using it in production."

Reply:
Works for me, thanks @curquiza 👌

Reply:
thats updated now in both place, resolving. thanks!